aspnetcore/blazor/fundamentals/additional-scenarios.md

```diff
@@ -218,6 +218,36 @@ Blazor.defaultReconnectionHandler._reconnectionDisplay =
 
 The placeholder `{ELEMENT ID}` is the ID of the HTML element to display.
 
+::: moniker range=">= aspnetcore-5.0"
+
+## Influence HTML `<head>` tag elements
+
+*This section applies to Blazor WebAssembly and Blazor Server.*
+
+When rendered, the `Title`, `Link`, and `Meta` components add or update data in the HTML `<head>` tag elements:
+
+```razor
+@using Microsoft.AspNetCore.Components.Web.Extensions
```

Comment:
This was just updated to `Microsoft.AspNetCore.Components.Web.Extensions.Head` (https://github.com/dotnet/aspnetcore/pull/23833 is now merged, so further changes like this won't occur 😃).

```suggestion
@using Microsoft.AspNetCore.Components.Web.Extensions.Head
```